### DISCUSSION TITLE
0.3.9

### DISCUSSION THREAD

**Discussion by mgmeyers** [mgmeyers/obsidian-kanban#104] [Releases]
- Adds ability to search Kanban boards using the default search hotkey

<img width="1064" alt="Screen Shot 2021-05-23 at 10 39 27 AM" src="https://user-images.githubusercontent.com/2694747/119270778-2806b380-bbb3-11eb-8de1-3f85b1c08cc0.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/mgmeyers/obsidian-kanban/releases/tag/0.3.9'>0.3.9</a>.</em>

**Comment by fuhadabdulla** [comment]
This is an awesome feature . 

Thank you so much
